sqlmesh/utils/jinja.py

```diff
@@ -358,6 +363,8 @@ def trim(
         for package, names in dependencies_by_package.items():
             result = result.merge(self._trim_macros(names, package))
 
+        result._trimmed = True
```

Comment:
This doesn't get serialized, so we won't know if it's trimmed after the deserialization. Is this intentional?

Reply:
yea, it's really only used during load from jinja disk